src/v2/view-builder/internals/BaseModel.js

```diff
@@ -28,22 +28,59 @@ const convertUiSchemaFieldToProp = (uiSchemaField) => {
   return { [uiSchemaField.name]: config };
 };
 
-const create = function (remediation = {}) {
-  const value = remediation.uiSchema;
-  // NOTE: consider moving logic to uiSchemaTransformer as well.
-  const props = _.chain(value)
-    .map(convertUiSchemaFieldToProp)
-    .reduce((init, field) => {
-      return Object.assign({}, init, field);
-    })
-    .value();
+const createPropsAndLocals = function (
```

Comment:
can you add an example involving subschema. Sample input output please

Reply:
I wish I could think of unit test earlier. ;)
Hopefully it's better than sample.